js/state/contentSettings.js

```diff
@@ -172,6 +172,16 @@ const getContentSettingsFromSiteSettings = (appState) => {
       addContentSettings(contentSettings.referer, hostPattern, '*', 'allow')
     }
   }
+
+  // temporarySiteSettings
```

Comment:
temporary site settings should only be added here if they are being sent to webcontents in a private session. cc @bridiver.